src/trans-netlist/unwind_netlist.cpp

```diff
@@ -141,21 +141,23 @@ Function: unwind_property
 
 \*******************************************************************/
 
-void unwind_property(
+bvt unwind_property(
   const netlistt::propertyt &property,
-  const bmc_mapt &bmc_map,
-  bvt &prop_bv)
+  const bmc_mapt &bmc_map)
 {
   PRECONDITION(std::holds_alternative<netlistt::Gpt>(property));
   auto property_node = std::get<netlistt::Gpt>(property).p;
 
+  bvt prop_bv;
   prop_bv.resize(bmc_map.timeframe_map.size());
```

Comment:
```suggestion
  bvt prop_bv{bmc_map.timeframe_map.size()};
```
or perhaps use `.reserve` instead of `.resize` and use `.push_back` in the loop below.